pypeit/fluxspec.py

```diff
@@ -444,22 +447,27 @@ def generate_sensfunc(self):
             return None
 
         ext_final = fits.getheader(self.par['std_file'], -1)
-        norder = ext_final['ECHORDER'] + 1
+        #norder = ext_final['ECHORDER'] + 1
+        norder = len(self.std)
 
         self.sens_dict = {}
         for iord in range(norder):
-            std_specobjs, std_header = load.load_specobjs(self.par['std_file'], order=iord)
-            std_idx = flux.find_standard(std_specobjs)
-            std = std_specobjs[std_idx]
+            #std_specobjs, std_header = load.load_specobjs(self.par['std_file'], order=iord)
+            #embed(header='447 of fluxspec')
+            #std_idx = flux.find_standard(std_specobjs)
+            std = self.std[iord] #std_specobjs[std_idx]
+
+            # THIS IS A CRAZY KLUDGE....
             try:
```

Comment:
wave_grid is going to be depcreataed, this can be removed.

Reply:
ok, edited.